ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-rollback-stable.yaml

```diff
@@ -0,0 +1,98 @@
+base_images:
+  ansible:
+    name: "4.12"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  cli-ocm:
+    name: cli-ocm
+    namespace: ci
+    tag: latest
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: openstack-installer
+  rosa-aws-cli:
+    name: rosa-aws-cli
+    namespace: ci
+    tag: latest
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.12"
+  tools:
+    name: "4.12"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.12"
+  oldest-supported-z-rollback:
+    release:
+      channel: candidate
+      version: 4.12.0
+  target:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.12"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: aws-ipi-byo-route53-upgrade-rollback-oldest-supported-f28
```

Comment:
Because we already place the job in a dedicated ci yam file, I think we do not need `upgrade-rollback-oldest-supported` any more, right?

Reply:
Yeah, I did think about dropping it. But we use it in the nightly rollback jobs, to make them consistent for easy management, I keep it on the stable rollback jobs. WDYT?

Reply:
> we use it in the nightly rollback jobs

The one in dev ci yaml, right?

Reply:
Or we can set the keyword in `zz_generated_metadata.variant`?

Reply:
Dropped `upgrade-rollback-oldest-supported` :)

Reply:
Nice update for `releases` configurations, I guess that resolved https://github.com/openshift/release/pull/48469/files#r1482354775

This comment is for ci jobs name.